src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(p => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter(p => p.sex === 'm');
+
+  const totalAge = men.reduce((prev, cur) => prev + (cur.died - cur.born), 0);
```

Comment:
Move this to separate function & reuse it
```suggestion
  const totalAge = getAvarageAge(men);
```